test/karma.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (config) {
+  config.set({
+    frameworks: ['mocha', 'chai'],
+    files: ['../dist/paste-html-to-markdown.js', 'test.js'],
```

Comment:
Aw do we have to compile before tests in karma?

That's going to make it a pain testing files :( 

I see you've got around it with https://github.com/kevindew/paste-html-to-govspeak/blob/a38c25d9cdf071abfe88c2672d3091025bff1883/src/main.js#L23-L25

But is there other approaches we could do? This was something that was nicer with jest

Reply:
I was planning to introduce karma so that, at some point, we can run automated cross-browser tests (e.g. via saucelabs). Happy to drop it if you think that's not the case. With the previous Jest setup we were unit testing the code, which worked fine without needing to attach them to window, but if we were going to introduce browser testing (e.g. with puppeteer) we would've end up doing the same thing.